src/main/java/appeng/parts/automation/UpgradeInventory.java

```diff
@@ -47,9 +47,10 @@
 
 	public UpgradeInventory( final IAEAppEngInventory parent, final int s )
 	{
-		super( null, s );
+		super( null, s, 1);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)